cmd/sebak/cmd/wallet/pay.go

```diff
@@ -0,0 +1,103 @@
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/network"
+	"github.com/stellar/go/keypair"
+
+	"log"
+)
+
+var (
+	flagNetworkID string = sebakcommon.GetENVValue("SEBAK_NETWORK_ID", "")
+	PayCmd        *cobra.Command
+	flagEndpoint  string
+)
+
+func init() {
+	PayCmd = &cobra.Command{
+		Use:   "pay <amount> <sender private key> <receiver pubkey>",
+		Short: "Send <amount> BOSCoin from one wallet to another",
+		Args:  cobra.ExactArgs(3),
+		Run: func(c *cobra.Command, args []string) {
+			var err error
+			var amount sebak.Amount
+			var sender keypair.KP
+			var receiver keypair.KP
+			var endpoint *sebakcommon.Endpoint
+
+			// Amount
+			if amount, err = common.ParseAmountFromString(args[0]); err != nil {
+				common.PrintFlagsError(c, "<amount>", err)
+			}
+
+			// Sender's private key
+			if sender, err = keypair.Parse(args[1]); err != nil {
+				common.PrintFlagsError(c, "<sender private key>", err)
+			} else if _, err = sender.Sign([]byte("witness")); err != nil {
+				common.PrintFlagsError(c, "<sender private key>", fmt.Errorf("Provided key is an address, not a private key"))
+			}
+
+			// Receiver's public key
+			if receiver, err = keypair.Parse(args[2]); err != nil {
+				common.PrintFlagsError(c, "<receiver public key>", err)
+			} else if _, err = receiver.Sign([]byte("witness")); err == nil {
+				common.PrintFlagsError(c, "<receiver public key>", fmt.Errorf("Provided key is a private key, not an address"))
+			}
+
+			// Check a network ID was provided
+			if len(flagNetworkID) == 0 {
+				common.PrintFlagsError(c, "--network-id", fmt.Errorf("A --network-id needs to be provided"))
+			}
+
+			if endpoint, err = sebakcommon.ParseNodeEndpoint(flagEndpoint); err != nil {
+				common.PrintFlagsError(c, "--endpoint", err)
+			}
+
+			// TODO: Validate input transaction (does the sender have enough money?)
+
+			// At the moment this is a rather crude implementation: There is no support for pooling of transaction,
+			// 1 operation == 1 transaction
+			var operation sebak.Operation
+			var tx sebak.Transaction
+			var connection *sebakcommon.HTTP2Client
+
+			body_payment := sebak.NewOperationBodyPayment(receiver.Address(), amount)
+			if operation, err = sebak.NewOperation(sebak.OperationPayment, body_payment); err != nil {
+				log.Panic("Error while creating operation: ", err)
+				os.Exit(1)
+			}
+
+			if tx, err = sebak.NewTransaction(sender.Address(), tx.NextCheckpoint(), operation); err != nil {
+				log.Panic("Error while creating transaction: ", err)
+				os.Exit(1)
+			}
+
+			tx.Sign(sender, []byte(flagNetworkID))
+
+			// Keep-alive ignores timeout/idle timeout
+			if connection, err = sebakcommon.NewHTTP2Client(0, 0, true); err != nil {
+				log.Panic("Error while creating network client: ", err)
+				os.Exit(1)
+			}
+
+			// Send request
+			client := sebaknetwork.NewHTTP2NetworkClient(endpoint, connection)
+			if err = client.SendMessage(tx); err != nil {
+				log.Panic("Network error: ", err)
```

Comment:
I thought It doesn't seem that it is the client issue. but the cli should return some useful messages. 
```
$~/sebak-wallet wallet pay 150000 SC7SB....JZK4VG4Y2U2J GBMDLGDH...SHSG3SBY2LBZL4VWM --endpoint=https://localhost:12345 --network-id=t1
anarch@vacuum-2 ~ // nothing result.. 
```
```
# in sebak node error msg. 
EROR[06-18|16:32:46] failed to handle message from client     module=sebak node=GCZU.7SRT error="{\"code\":132,\"message\":\"invalid checkpoint found\"}"
```

Reply:
I followed the [rule of silence](http://www.linfo.org/rule_of_silence.html).
In this case, the error is benign, because it just reject a duplicated broadcast. I don't think there's any reasonable way to get feedback from a transaction anyway.

Reply:
@Geod24 @anarcher if successfully done, it can be reasonable without success message, but if error occurred, the panic message looks like a bug. It will be better to provide more organized message in error.